frontend/src/components/ImageInfoCard/ImageInfoCard.tsx

```diff
@@ -12,7 +13,9 @@ function ImageInfoCard({ imageUrl, title, description }: ImageInfoCardProps) {
       <Image src={imageUrl} alt={`${title} 뉴스레터 이미지`} />
       <InfoBox>
         <Title>{title}</Title>
-        <Description>{description}</Description>
+        <Description>
+          {description.length === 0 ? title : description}
```

Comment:
p4) 이렇게 하면 코드를 조금 더 간결하게 표현할 수 있을 것 같아요!

```suggestion
          {description || title}
```

Reply:
오 좋아요!